client/src/botany_client/client.py

```diff
@@ -147,6 +148,16 @@ def wrapped(board, move_list, token, state):
     if opcode_limit == 0:
         opcode_limit = None
 
+    #Initiate the move list parameter
+    if move_list is None:
+        move_list = []
+    else:
+        assert move_list.isdigit(), "move-list should only consist of integers"
```

Comment:
We don't want this `assert`, as it is specific to the rules in `botany_connectfour.game`, and other games might allow non-digits as moves.